indico/web/assets/bundles.py

```diff
@@ -172,7 +172,8 @@ def rjs_bundle(name, *files):
                'palettepicker.js',
                'itempicker.js',
                'sortablelist.js',
-               'categorynavigator.js'))
+               'categorynavigator.js',
+               '../modules/eventcreation.js'))
```

Comment:
It's needed on all pages. But yeah I could add it as a module JS and always include it